FogCreek/Glitch-Community
# Foggy leopon: add aria labels to private project signifiers

## Links
* Remix link: https://foggy-leopon.glitch.me
* Manuscript link: https://glitch.manuscript.com/f/cases/3328729/Lock-toggle-to-make-private-on-a-project-page-is-missing-alt-text

## Changes:
* add aria-label to private project lock icon and toggle
* use project badge component in project items

## How To Test:
* log into https://foggy-leopon.glitch.me
* visit https://foggy-leopon.glitch.me/~support-poster
* verify that the private project icon has a label, even when its not focused
* visit one of your own private projects
* verify that the private project toggle has a label, even when its not focused
